samples/dash-if-reference-player/app/main.js

```diff
@@ -946,6 +946,24 @@ app.controller('DashController', function ($scope, sources, contributors, dashif
             item.url = vars.source;
         }
 
+        // If supply widevine= parameter in the query string then set up the DRM protection to playback Widevine
+        if (vars && vars.hasOwnProperty('widevine')) {
+            $scope.drmLicenseURL = vars.widevine;
+            if ($scope.drmLicenseURL.indexOf("%")) {
+                $scope.drmLicenseURL = unescape($scope.drmLicenseURL);
+            }
+            $scope.drmKeySystem = $scope.drmKeySystems[0];
```

Comment:
I like this feature. @kevleyski , instead of hardcoding the index of the drm system, I think it would be better to find in the DRM for the one that makes sense for the option selected by the user (using $scope.drmKeySystems.indexOf).

Thanks!

Reply:
Yeah I was trying to not duplicate the constants, its easier t read however with the last edit if you are happy with it

Ideally need a way to reference ProtectionConstants instead, but as the code has this string ins several places, one more I guess doesn't matter

Or some indexOf Knuth–Morris–Pratt string-searching algorithm to find widevine, but thats just a bit daft